digdag-core/src/main/java/io/digdag/core/database/DatabaseTaskQueueConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.digdag.core.database;
+
+import io.digdag.client.config.Config;
+import com.google.inject.Inject;
+
+public class DatabaseTaskQueueConfig
+{
+    private final int defaultMaxConcurrency;
+
+    @Inject
+    public DatabaseTaskQueueConfig(Config systemConfig)
+    {
+        this.defaultMaxConcurrency = systemConfig.get("queue.db.max_concurrency", int.class, Integer.MAX_VALUE);
+    }
+
+    public int getSiteMaxConcurrency(int siteId)
```

Comment:
It's better to have but not implemented yet because I think few people use multiple sites.
On a hosted environment (TD), application should override this class using DI so that it gets max_concurrency from a remote database.